server/server.py

```diff
@@ -105,7 +118,7 @@ def parseArgs():
 	#check if we have the private key
 	if not isfile("./python.pem"):
 		logger.critical("[-] Missing private key, Exiting")
-		exit(-2)
+		exit(3)
```

Comment:
Nice catch, I originally had them all as negatives, but since the return is a 8 bit unsigned int, it made it really hard to figure out where the exit was coming from. :taco: for you!